src/kartograf/filters/element_change.py

```diff
@@ -0,0 +1,40 @@
+import logging
+from rdkit import Chem
+
+logger = logging.getLogger(__name__)
```

Comment:
maybe we could name the loggers for filters: Kartograf_Atom_Filter or something like that?

Reply:
I think that is what I wanted ^^ what do you think @richardjgowers

Reply:
Yeah I think using `__name__` here makes sense so we can filter the logging as you would expect